CharlieBackend.Data/Repositories/Impl/StudentGroupRepository.cs

```diff
@@ -90,5 +91,17 @@ public new Task<StudentGroup> GetByIdAsync(long id)
                     .Include(group => group.MentorsOfStudentGroups)
                     .FirstOrDefaultAsync(group => group.Id == id);
         }
+
+        public async Task<List<MentorStudyGroupsDto>> GetMentorStudyGroups(long id)
+        {
+            return await _applicationContext.StudentGroups
+                    .Include(group => group.MentorsOfStudentGroups)
+                    .Where(x => x.MentorsOfStudentGroups.Any(x => x.MentorId == id))
```

Comment:
Any checks for active/inactive group, course?

Reply:
Is it needed?